src/pages/distroSettings/tabs/GeneralTab/getFormSchema.ts

```diff
@@ -1,9 +1,12 @@
 import { css } from "@emotion/react";
 import { GetFormSchema } from "components/SpruceForm";
 import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
+import { DeleteDistro } from "./DeleteDistro";
 
 export const getFormSchema = (): ReturnType<GetFormSchema> => ({
-  fields: {},
+  fields: {
+    deleteDistro: DeleteDistro,
+  },
```

Comment:
Do you think there's any benefit to including this section in the form? It seems pretty doable to add a separate Card component below the SpruceForm. On the one hand, including it in the form gets you some built in formatting rules; on the other, I'm inclined to keep the form minimal for clarity and performance reasons.